WordPress/Classes/Utility/Universal Links/Routes+Reader.swift

```diff
@@ -62,7 +62,8 @@ extension ReaderRoute: Route {
 
 extension ReaderRoute: NavigationAction {
     func perform(_ values: [String: String], source: UIViewController? = nil, router: LinkRouter) {
-        guard let coordinator = WPTabBarController.sharedInstance().readerCoordinator else {
+        guard let tabBarController = RootViewControllerCoordinator.sharedPresenter.rootViewController as? WPTabBarController,
+              let coordinator = tabBarController.readerCoordinator else {
```

Comment:
Could `readerCoordinator` be part of `RootViewPresenter` protocol? We already have `Reader` section there. It could return `nil` same way as `readerTabViewController`.

Reply:
Done in 5a442959943a47db48d6f282f6c4a8a6937e6764 👍